bin/mouse-preview

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+
+from mouse.preview import main
+
+if __name__ == "__main__":
+   import sys
+   
```

Comment:
Trailing whitespace here. Keep the newline, but eliminate extra spaces. This is true of the other binaries too.